src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.annotation.Nonnull;
 
+import io.github.thebusybiscuit.slimefun4.implementation.items.armor.ElytraCap;
```

Comment:
Why is this import up here btw and not with the rest?

Reply:
dunno

Reply:
Then move it back down to the rest?